### PR TITLE
Improved HGM Circles a little bit

### PR DESCRIPTION
**Pull request by tomgrubbmath**
Deals with part of Issue #3406 . Added a frame around HGM circles to prevent skewing of the aspect ratio, thickened the circle, made the background transparent, and deepened the colors. Two examples below, left is new, right is old. It is easy to change color schemes if anyone has preferences:

image

image

### CODE REVIEW

File: lmfdb/hypergm/plot.py [hunk 1]
```diff
@@ -47,7 +47,9 @@ def piecewise_linear_image(A,B):
     
 def circle_image(A,B):
     G = Graphics()
-    G += circle((0,0), 1 , color = 'grey')
+    G += circle((0,0), 1 , color = 'black',thickness = 3)
+    backgroundColor = (227/255,242/255,253/255) #This is the RGB color (in Sage format) of the properties bar (found in style.css). Making the framing circle this color makes it invisible on the page.
```
Comment (edgarcosta) [inline lmfdb/hypergm/plot.py:51]: `(227/255,242/255,253/255, 0)` might work better and work independently of the current theme

**Comment by edgarcosta** [comment]
Overall this is a significant improvement and I'm happy to merge it.

File: lmfdb/hypergm/plot.py (hunk 1 above)
Comment (tomgrubbmath) [reply lmfdb/hypergm/plot.py:51]: Good idea; I'll edit it in and test it out to make sure.

Comment (tomgrubbmath) [reply lmfdb/hypergm/plot.py]: Turns out there is a builtin transparency parameter ("alpha") for graphics that I missed. The frame should now be invisible, regardless of background theme.

**Comment by edgarcosta** [comment]
Once the tests pass I will merge

**Pull request merged**